### ISSUE TITLE
Can hermes work with network tab in chrome dev tools?

### ISSUE THREAD

**Issue by matthewmturner** [facebook/hermes#959]
## Problem

From the title - is it possible to make Hermes work with the network tab in chrome dev tools when debugging react native app?  That would significantly improve the debugging experience as i currently need to use other tools for checking networking activity.

## Solution

I would like to see all my apps network activity in the network tab of chrome dev tools

**Comment by mattbfb** [comment]
tl;dr: I believe the end result you're after is possible, but this information would be generated outside of Hermes. Inquiring on the React Native repository (https://github.com/facebook/react-native) may be a good starting point.

Details:

Hermes really only has visibility into JavaScript-related information (in Chrome Dev Tools terms, this is presented in tabs like "Sources" and "Memory"). The Hermes runtime doesn't perform network operations, so it can't report them.

Other tabs like "Elements" and "Network" display information from other facets of the application outside of the JavaScript runtime, and would be populated by whatever part of the application manages elements or performs network activity. I'm not familiar with how these parts function in React Native, so I'll have to defer to them for more details.

**Comment by matthewmturner** [comment]
@mattbfb thanks! will raise there and close this for now.  appreciate the quick response.
